templates/_partials/fields/select.html.twig

```diff
@@ -15,6 +15,12 @@
     {% set multiple = 'false' %}
 {% endif %}
 
+{% if field.allowempty is defined %}
```

Comment:
Shouldn't it be `if allowempty is not defined` ?

Reply:
Yeah, this logic is wonky.. Fixed in latest commit! 👍